main.go

```diff
@@ -194,71 +174,51 @@ func main() {
 	// Initialize event recorder.
 	record.InitFromRecorder(mgr.GetEventRecorderFor("observability-operator"))
 
-	var managementCluster common.ManagementCluster = common.ManagementCluster{
-		BaseDomain: managementClusterBaseDomain,
-		Customer:   managementClusterCustomer,
-		InsecureCA: managementClusterInsecureCA,
-		Name:       managementClusterName,
-		Pipeline:   managementClusterPipeline,
-		Region:     managementClusterRegion,
-	}
-
 	var opsgenieApiKey = os.Getenv(opsgenieApiKeyEnvVar)
 	if opsgenieApiKey == "" {
 		setupLog.Error(nil, fmt.Sprintf("environment variable %s not set", opsgenieApiKeyEnvVar))
 		os.Exit(1)
 	}
 
-	heartbeatRepository, err := heartbeat.NewOpsgenieHeartbeatRepository(opsgenieApiKey, managementCluster)
+	heartbeatRepository, err := heartbeat.NewOpsgenieHeartbeatRepository(opsgenieApiKey, conf.ManagementCluster)
```

Comment:
Any reason why you do not add env variables in the config like the opsgenie api key?

Reply:
I am trying to not change everything at once.